tests/snapshots/assets__check_file@packages-codly.typ-120.snap

```diff
@@ -925,15 +932,24 @@ snapshot_kind: text
     0pt
   }
 
-  let annot-width = annot-bodies-width.fold(
-    0pt,
-    (a, b) => calc.max(a, b),
-  ) + padding.left + padding.right + lr-width
+  let annot-width = (
+    annot-bodies-width.fold(
+      0pt,
+      (a, b) => calc.max(a, b),
+    )
+      + padding.left
+      + padding.right
+      + lr-width
+  )
 
   // Get the height of an individual line.
-  let line-height = measure(
-    raw.line(1, 1, "Hello, World!", [Hello, World!]),
-  ).height + padding.top + padding.bottom
+  let line-height = (
+    measure(
+      raw.line(1, 1, "Hello, World!", [Hello, World!]),
+    ).height
+      + padding.top
+      + padding.bottom
+  )
 
```

Comment:
same as above but due to longer chain, this looks better. We may refer to rustfmt for these cases